packages/service/ni_measurement_plugin_sdk_service/measurement/service.py

```diff
@@ -424,6 +424,10 @@ def configuration(
                 "DataType.PinArray1D is deprecated. Use DataType.IOResourceArray1D instead.",
                 DeprecationWarning,
             )
+        if type == DataType.Double2DArray:
+            raise ValueError(
+                "DataType.Double2DArray is not supported for configuration parameters."
```

Comment:
That's not the only unsupported configuration data type, is it?
```suggestion
        if type in [DataType.Double2DArray, DataType.DoubleXYData, DataType.DoubleXYData1D]:
            raise ValueError(
                f"{type} is not supported for configuration parameters."
```